install_requirements.py

```diff
@@ -139,6 +152,24 @@ def install_requirements(use_pytorch_nightly):
     )
 
 
+# Prebuilt wheels for Intel macOS are no longer available on PyPI; users must compile from source.
+# PyTorch stopped building macOS x86_64 binaries since version 2.3.0 (January 2024).
+def is_intel_mac_os():
+    # Returns True if running on Intel macOS
+    if platform.system().lower() == "darwin" and platform.machine().lower() in (
+        "x86",
+        "x86_64",
+        "i386",
+    ):
+        print(
+            "ERROR: Prebuilt PyTorch wheels are no longer available for Intel-based macOS.\n"
+            "Please build from source by following https://pytorch.org/executorch/0.6/using-executorch-building-from-source.html",
```

Comment:
https://pytorch.org/executorch/0.6/ -> change this to https://pytorch.org/executorch/main/